README.md

```diff
@@ -64,7 +64,7 @@ container images and config changes are propagated to the cluster.
 |[Under Armour](https://www.underarmour.com)|[VSHN](https://vshn.ch)|[Walmart Chile](https://www.walmartchile.cl)
 |[Weave Cloud](https://cloud.weave.works)|[Workable](https://www.workable.com)|[Workarea](https://www.workarea.com)
 |[Working Group Two](https://wgtwo.com)|[Yusofleet](https://yusofleet.com)|[Yad2](https://yad2.co.il)
-|[Zaaksysteem](https://zaaksysteem.nl)|
+|[Zaaksysteem](https://zaaksysteem.nl)|[Chattigo](https://chattigo.com)
```

Comment:
Can you please move the added entry so alphabetical order is maintained?

Reply:
Yeah! Just pushed it.